src/plugins/trivia.js

```diff
@@ -34,7 +34,10 @@ async function skip(message) {
   );
 
   if (skipCount >= 2) {
+
+    await db.resetTriviaHintVoteCount();
```

Comment:
Do these depend on each other? Could they be run in parallel?

Reply:
sure

Reply:
is paralleling a javascript thing? cause all I can gather is just puting them in a function call

Reply:
```
await Promise.all([db.resetTriviaHintVoteCount(), db.resetTriviaSkip(), db.increaseTriviaHintLevel()])
```

This will run the three functions concurrently. Order of execution is not guaranteed.